src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -96,7 +96,7 @@ public class NodeSettings : IDisposable
         ///   name would be determined. In this case we first need to determine the network.
         /// </remarks>
         public NodeSettings(Network network = null, ProtocolVersion protocolVersion = SupportedProtocolVersion,
-            string agent = "StratisBitcoin", string[] args = null, NetworksSelector networksSelector = null)
+            string agent = "Stratis", string[] args = null, NetworksSelector networksSelector = null)
```

Comment:
Maybe StratisFN will be better to avoid mixing stratis full node platform with stratis coin.

Reply:
We thought about StratisNode yesterday but in the end we thought Stratis was shorter (it is) and cuter.The coin is called strat, or $STRAT on twitter lol.